pkg/apis/kubeflow.org/v1/pytorch_types.go

```diff
@@ -84,6 +84,8 @@ type PyTorchJobSpec struct {
 	//     "Worker": PyTorchReplicaSpec,
 	//   }
 	PyTorchReplicaSpecs map[ReplicaType]*ReplicaSpec `json:"pytorchReplicaSpecs"`
+	// Number of workers per node
```

Comment:
Should we indicate when this parameter should be used ?
E.g. `when each PyTorch replica uses more than one GPU`.

Reply:
I've made `nprocPerNode` as string type which is exactly consistent with pytorch itself, so I think it is OK to let user check the meaning from the pytorch doc.

Further, the default value is set to "auto" and user do not need to care about it in most case even with GPUs resources declaration.

Reply:
@kuizhiqing Should we add the link to PyTorch doc in the APIs so user knows how this value can be set ?

Reply:
Done